circle.yml

```diff
@@ -32,6 +33,7 @@ notify:
 test:
   override:
     - bundle exec rake lint:bash
+    - cd /home/ubuntu/.go_workspace/src/github.com/Originate/gt && gometalinter.v1 --deadline=20s cmd/
```

Comment:
You're most likely going to want to pick which linters you run, some of them are quite pedantic. I believe `gometalinter` also needs the code to be built before running (`go install ./...`). I typically use something like this:
```bash
gometalinter --deadline 300s \
  -E unused -E misspell \
  --tests --enable-gc --aggregate \ 
  $(glide novendor)
```